data/mods/Magiclysm/enchanted/enchanted_melee.json

```diff
@@ -754,5 +754,176 @@
     "name": "dao +2",
     "proportional": { "price": 6.0, "bashing": 1.2, "cutting": 1.2, "weight": 0.8 },
     "relative": { "to_hit": 2 }
+  },
+  {
+    "id": "rune_biomancer_weapon",
+    "type": "GENERIC",
+    "name": "Biomancer spear",
+    "plural": "Biomancer spears",
```

Comment:
```suggestion
```

Reply:
As I have recently discovered thanks to KorG, you don't need to do this at all, but even if you did, it's "name_plural".

Reply:
Actually the newest format is "name": { "str": "xxx", "str_pl", "xxxs" }, after the recent migration to use `translation` for item/monster names. "name_plural" is still supported though.

Reply:
Ugh. Great, now I have even more work to do in updating..